components/modals/AgendaModal.tsx

```diff
@@ -0,0 +1,46 @@
+import { Modal } from "@mui/material";
+import { classConfigRecurrentId, sitClassRecurrentId } from "../../lib/iBooking";
+import React, { Dispatch, SetStateAction } from "react";
+import Agenda from "../Agenda";
+import { UserConfig } from "../../types/rezervoTypes";
+import { SitClass } from "../../types/sitTypes";
+
+const AgendaModal = ({
+    open,
+    setOpen,
+    userConfig,
+    classes,
+    selectedClassIds,
+    onInfo,
+    onSelectedChanged,
+}: {
+    open: boolean;
+    setOpen: Dispatch<SetStateAction<boolean>>;
+    userConfig: UserConfig | undefined;
+    classes: SitClass[];
+    selectedClassIds: string[] | null;
+    onInfo: Dispatch<SetStateAction<SitClass | null>>;
+    // eslint-disable-next-line no-unused-vars
+    onSelectedChanged: (classId: string, selected: boolean) => void;
+}) => {
+    if (!userConfig?.classes) {
+        return <></>;
+    }
```

Comment:
I would like this check to be inside `Modal` (like `ClassInfoModal`), so that it will always open and close (but be blank if classes are missing)

Reply:
Sure thing ✅